### ISSUE TITLE
ERROR: error code -11 when Train a detection model using DIGITS (with GPU)

### ISSUE THREAD

**Issue by alex13701122a** [NVIDIA/DIGITS#1448]
Hi everyone
I am following the tutorial on 'Using DIGITS to train Object Detection network' [https://github.com/NVIDIA/DIGITS/tree/master/examples/object-detection](url)
but after Epoch 2 I reach ERROR: error code -11.  [caffe_output.log.txt](https://github.com/NVIDIA/DIGITS/files/763016/caffe_output.log.txt)

I changed Batch size and Base Learning Rate and this error occur after Epoch 5 or later but I couldn't solve this error.
My DIGITS version is 5.1-dev, Caffe version is 0.15.14 and my GPU is NVIDIA Gforce GTX1060.
Not that this error occurs when I try to create a detection model. 
When I use DIGITS for classification, my classification models had been created correctly.

thanks.

**Comment by lukeyeager** [comment]
Looks like a possible OpenCV error. Where did you get OpenCV from?

**Comment by alexBatSharif** [comment]
I was installed Opencv3.2 using Cmake in Ubuntu 1404.
after occur this error I think maybe Opencv or Caffe has installed incorrectly, so I deleted Caffe folder(where I installed it) to remove it and reinstalled it using https://github.com/NVIDIA/DIGITS/blob/digits-4.0/docs/BuildCaffe.md
I see using Install some dependencies with Deb packages command I install Opencv 2.4.9.
now I have both of them.

**Comment by lukeyeager** [comment]
I don't fully understand your comment. Did you figure it out? If so, you can close this issue.

**Comment by alexBatSharif** [comment]
I don't know.
when I searched for error code -11 in DIGITS I could not find any useful comments.

**Comment by lukeyeager** [comment]
OpenCV can be difficult to uninstall fully. You might want to consider using nvidia-docker?
https://github.com/NVIDIA/nvidia-docker/wiki/DIGITS
